lib/puppet/provider/rabbitmq_user/rabbitmqctl.rb

```diff
@@ -1,7 +1,9 @@
-require 'puppet'
-require 'set'
-require File.expand_path(File.join(File.dirname(__FILE__), '..', 'rabbitmqctl'))
-Puppet::Type.type(:rabbitmq_user).provide(:rabbitmqctl, :parent => Puppet::Provider::Rabbitmqctl) do
+require 'puppet/provider/rabbitmqctl'
+
+Puppet::Type.type(:rabbitmq_user).provide(
+  :rabbitmqctl,
+  :parent => Puppet::Provider::Rabbitmqctl
+) do
 
   if Puppet::PUPPETVERSION.to_f < 3
```

Comment:
seems like the bits related to Puppet <3 or <= 3 can all go away since the module is 4.8 and higher.